sources/osg/CullVisitor.js

```diff
@@ -50,6 +50,9 @@ var CullVisitor = function () {
     this._reserveLeafStack = [ new RenderLeaf() ];
     this._reserveLeafStack.current = 0;
 
+    this._reserveCullSettingsStack = [ new CullSettings() ];
```

Comment:
yep in the meantime split the current in another variable, and add a task to make a 'generic' stack object